### PR TITLE
Obstacle removal

### PR DESCRIPTION
**Pull request by rahul-krupani**
Ep_time removal

### CODE REVIEW

File: gym_art/quadrotor_multi/octomap_creation.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def generate_obstacles(self, num_obstacles=0, start_point=np.array([-3.0, -2.0,
         self.end_range = np.array([end_point[:2] + self.init_box[0][:2], end_point[:2] + self.init_box[1][:2]])
         pos_z = 0.5 * self.room_dims[2]
         for i in range(num_obstacles):
-            for regen_id in range(20):
+            for regen_id in range(30):
                 y_gaussian_scale = self.y_gaussian_generation(regen_id=regen_id)
```
Comment (Zhehui-Huang) [inline gym_art/quadrotor_multi/octomap_creation.py]: Why change this to 30?

File: gym_art/quadrotor_multi/quadrotor_multi.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 from gym_art.quadrotor_multi.quadrotor_single import GRAV, QuadrotorSingle
 from gym_art.quadrotor_multi.quadrotor_multi_visualization import Quadrotor3DSceneMulti
-from gym_art.quadrotor_multi.quad_scenarios import create_scenario
+from gym_art.quadrotor_multi.scenarios.mix import create_scenario
```
Comment (Zhehui-Huang) [inline gym_art/quadrotor_multi/quadrotor_multi.py:17]: Put create_scenario into utils.py

File: gym_art/quadrotor_multi/quadrotor_single.py [hunk 3]
```diff
@@ -20,6 +20,7 @@
 import argparse
 import logging
 import sys
+import time
```
Comment (Zhehui-Huang) [inline gym_art/quadrotor_multi/quadrotor_single.py]: We didn't use `import time`

**Review by Zhehui-Huang — CHANGES_REQUESTED** [review]
Please check my comments. Besides, please also pull my latest commit as well. Use following command.

1) git remote add upstream    
 https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git
 2) git fetch upstream
 3) git checkout BRANCH_NAME (This is branch on your side, for you, it is obstacle-removal)
4) git merge upstream/UPSTREAM_BRANCH_NAME (obstacle_design)

File: gym_art/quadrotor_multi/octomap_creation.py (hunk 1 above)
Comment (rahul-krupani) [reply gym_art/quadrotor_multi/octomap_creation.py]: Was just debugging, fixed

**Review by Zhehui-Huang — APPROVED** [review]
Although there are still some issues in your PR, but not a big deal. I will change that for you.
For example, in mix.py, you need to remove all unused class. Please DO CLEAN your code next time before creating a PR.

**Pull request merged**